src/measure_theory/measurable_space.lean

```diff
@@ -729,19 +728,14 @@ end
 
 end
 
-section fintype
-
-local attribute [instance] fintype.to_encodable
-
-lemma measurable_set.pi_fintype [fintype δ] {s : set δ} {t : Π i, set (π i)}
+lemma measurable_set.pi_countable [countable δ] {s : set δ} {t : Π i, set (π i)}
   (ht : ∀ i ∈ s, measurable_set (t i)) : measurable_set (pi s t) :=
 measurable_set.pi (to_countable _) ht
```

Comment:
Should we drop this lemma? `measurable_set.pi_countable H` is not much shorter than `measurable_set.pi s.to_countable H`.